practice-one/src/Components/Common/Card.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Card = ({
+  maxW = 'xl',
+  name = '',
+  size = 'medium',
+  variant = 'primary',
+  children,
+  onChange = () => {},
+}) => {
+  const commonClasses = 'bg-white ';
+  const sizeClasses = {
+    small: 'p-4',
+    medium: 'p-6',
+    large: 'p-8',
+  };
+  const variantClasses = {
+    primary: 'border-none',
+    secondary: 'border-secondary',
+    success: 'border-green-500',
+    danger: 'border-red-500',
+    warning: 'border-yellow-500',
+  };
+
+  const cardClasses = `${commonClasses} ${sizeClasses[size]} ${variantClasses[variant]}`;
+
+  return (
+    <div className={`max-w-${maxW} mx-auto ${cardClasses}`} onChange={onChange}>
+      {name && <h2 className="text-sm font-bold mb-4">{name}</h2>}

```

Comment:
I don't see any example using name. Please help check

Reply:
Done, I removed this